src/query/storage/promremote/storage.go

```diff
@@ -543,9 +543,11 @@ func (p *promStorage) write(
 			err = nil
 			break
 		}
-		p.retryWrites.Inc(1)
-		time.Sleep(backoff)
-		backoff *= 2
+		if i > 0 { // only do backoff if we are going to retry
+			p.retryWrites.Inc(1)
+			time.Sleep(backoff)
```

Comment:
Can we add a random jitter to avoid many goroutine retrying at the same time?
```
time.Sleep(backoff + rand.Intn(100) * time.Millisecond)
```

Reply:
the remote write should be distributed and not fire at the same time already and chunked by sample size